src/DotNetLightning.Core/Channel/CommitmentToLocalExtension.fs

```diff
@@ -96,6 +98,12 @@ type internal CommitmentToLocalExtension() =
             // could not be generated.
             match pubKey with
             | null -> null
+            | _ when pubKey = parameters.RevocationPubKey.RawPubKey() ->
+                let revocationSig = signer.Sign (parameters.RevocationPubKey.RawPubKey())
+                Script [
```

Comment:
We don't have a choice. This is overriding `NBitcoin`'s `BuilderExtension.GenerateScriptSig` method which is supposed to return `null` if it can't generate the script sig.

Reply:
let's put a comment that explains this, just above the `null` element